examples/bpi-monolith-test-suite/bpi/components/remotes/gnarkwasm.remote.ts

```diff
@@ -0,0 +1,7 @@
+import { internalAxios } from './internal.axios';
+import { privateInput } from './privateInput';
+
+export const checkValidity = async (input: privateInput) => {
+    const response = await internalAxios.post('/', input)
```

Comment:
instead of making internalAxios like this, i think it would be cleaner to have services that do http calls and you call services methods and you move api url to env variable and end up with something like
service:

function serviceMethod {
axios.post(ENV.api, ...)

}

Reply:
This file removed as unused code